js/components/Checkbox.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
```

Comment:
Consistency is important in software development as it will make our source easier to read and understand for future developers.  So that's why I say if we bring it into our version control we should make it consistent with out current style and ES6 syntax.